rush.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.109.1",
```

Comment:
revert change to rush.json, this is being handled in its own PR #3700

Reply:
right i was going to remove this

Reply:
Looks like the issue may not be the rush update after all.
There's an issue with `rush switch-flavor:beta-release` and it looks like the only real difference between 109 and previous rush versions is that the error is now reported differently.

Reply:
I have reverted the 5.109 change so we should be good here